### ISSUE TITLE
Problem: Cannot find rdss-archivematica-shib-sp-proxy image when deploying

### ISSUE THREAD

**Issue by mamedin** [JiscRDSS/rdss-archivematica#218]
When deploying with:

```make all ENV=dev SHIBBOLETH_CONFIG=archivematica``` 

I got this error:

```
Pulling shib-sp-proxy (rdss-archivematica-shib-sp-proxy:latest)...
ERROR: pull access denied for rdss-archivematica-shib-sp-proxy, repository does not exist or may require 'docker login'
Makefile:180: recipe for target 'up' failed
make: *** [up] Error 1
```

**Comment by mamedin** [comment]
Fixed adding `${REGISTRY}` in image section at file:

```
maml@cabesa:~/rdss-archivematica/compose$ git diff docker-compose.shib-local.yml
diff --git a/compose/docker-compose.shib-local.yml b/compose/docker-compose.shib-local.yml
index 724c45f..4cd5bbe 100644
--- a/compose/docker-compose.shib-local.yml
+++ b/compose/docker-compose.shib-local.yml
@@ -12,7 +12,7 @@ services:
 
   # Shibboleth IdP service for the domain
   idp:
-    image: "arkivum/shibboleth-idp:${DOMAIN_NAME}"
+    image: "${REGISTRY}arkivum/shibboleth-idp:${DOMAIN_NAME}"
     build:
       context: "./shib-local/build/idp/"
     hostname: "idp"
```

**Comment by sevein** [comment]
Confirmed, this worked for me.

**Comment by sevein** [comment]
Closed by #222.
